common/interfaces/user_interface/ui_multi_instruction.c

```diff
@@ -383,3 +406,37 @@ void multi_instruction_init(const char **arr, const uint8_t count, const uint16_
 
     multi_instruction_create();
 }
+
+void multi_instruction_with_image_init(instruction_content_t content[], const uint8_t count, const uint16_t delay_in_ms, const bool destruct_on_click)
+{
+
+    data = NULL;
+    data = malloc(sizeof(struct Multi_Instruction_Data));
+    obj = NULL;
+    obj = malloc(sizeof(struct Multi_Instruction_Object));
+
+
+    for (uint8_t i = 0; i < count; i++)
+    {
+        memcpy(&(data->instruction_content[i]), &content[i], sizeof(instruction_content_t));
+        if(content[i].img != NULL){
```

Comment:
please add `else` case here

Reply:
Don't need this check anymore, removed the variable as it was not required anymore.